helix-redirects.yaml

```diff
@@ -1,5 +1,5 @@
 redirects:
-  - https://adobe.sharepoint.com/:x:/s/TheBlog/EaikC7JAUNpAoZy604FUP7YB2oTdxZVtrWNihbJ6k-BCnA?email=&e=Nd2OF4
+  - https://master--theblog--adobe.hlx.page/redirects.json
```

Comment:
Are you sure that this works?

Reply:
Nevermind, I forgot about https://github.com/adobe/helix-shared/pull/438

Reply:
it should :-) but I can test it....

Reply:
hmmm...it doesn't work and helix-redirect is not very verbose :-(

Reply:
https://github.com/adobe/helix-shared/issues/445

Reply:
ok works:

```
$ curl -sD - 'https://lqmig3v5eb.execute-api.us-east-1.amazonaws.com/helix-services/redirect/v1?owner=tripodsan&repo=helix-pages-test&ref=master&path=/tag/coronavirus/'
HTTP/2 301
date: Wed, 03 Mar 2021 11:39:15 GMT
content-type: text/plain; charset=utf-8
content-length: 17
location: https://blog.adobe.com/en/topics/covid-19.html
cache-control: max-age=30000000
apigw-requestid: bm9G7h-jIAMEVlw=

moved permanently
```